bundles/org.openhab.binding.volvooncall/src/main/resources/OH-INF/thing/vehicle.xml

```diff
@@ -264,8 +266,8 @@
 	<channel-type id="fuelConsumption" advanced="true">
 		<item-type>Number</item-type>
 		<label>Average Consumption</label>
-		<description>Indicates the average fuel consumption in L/100km</description>
-		<state pattern="%.1f L/100km" readOnly="true"></state>
+		<description>Indicates the average fuel consumption in l/100km</description>
+		<state pattern="%.1f l/100km" readOnly="true"></state>
```

Comment:
Why are the units in volume/distance instance of distance/volume? Is this just a Volvo thing?

Reply:
Not especially Volvo, it more European to indicate fuel consumption / 100km - I saw that in US (or the rest of the world) it's different.